ietf/templates/nomcom/view_feedback.html

```diff
@@ -54,6 +54,14 @@ <h3 class="mt-5" id="declined">Declined each nominated position</h3>
                     </tr>
                 {% endfor %}
             </tbody>
+	    <tfoot>
+		<td></td>
+		<td></td>
+		<th>Totals</h></th>
+		{% for ft in nominee_feedback_types %}
+		  <td>{{ feedback_totals|lookup:ft }}</td>
```

Comment:
This line doesn't seem to work :(

Reply:
Of course that fixed it.  I missed the deref in the previous `for ft in` loops :(
Still need to write a test.